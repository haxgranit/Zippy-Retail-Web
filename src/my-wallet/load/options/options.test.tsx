import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import Options from './Options';

const ReactTestRenderer = require('react-test-renderer');

describe('TransactionComplete Component', () => {
  it('matches the snapshot', () => {
    const fundingSources = [{ id: 1, isDefault: true, bankAccount: { institutionNumber: '123', transitNumber: '12345', accountNumber: '1234567890' } }, {
      id: 2,
      displayName: 'My Credit Card',
      isDefault: false,
      paymentCard: {
        name: 'John Doe', number: '1234456789012345', cvv: '123', expireDate: '2024-01-01T00:00:00',
      },
    }];
    const tree = ReactTestRenderer.create(
      <BrowserRouter>
        <Options
          defaultTitle="Select funding source"
          fundingSources={fundingSources}
          onChange={() => {}}
        />
      </BrowserRouter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
