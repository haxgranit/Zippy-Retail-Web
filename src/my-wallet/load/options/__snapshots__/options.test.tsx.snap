// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionComplete Component matches the snapshot 1`] = `
<div
  className="position-relative"
>
  <div
    className="dropbtn1"
    onClick={[Function]}
    onKeyDown={[Function]}
    role="button"
    tabIndex={0}
  >
    Select funding source
     
    <img
      alt=""
      src="https://dash.com.sg/assets/images/icons/arrow--down--grey.svg"
      width={30}
    />
  </div>
</div>
`;
